src/client/routes/login/PasswordLogin.tsx

```diff
@@ -29,21 +26,23 @@ interface Props {}
  */
 export const validateAndSubmitLogin = (
 	username: string,
-	password: string,
-	setInvalidFn: React.Dispatch<React.SetStateAction<boolean>>
+	password: string
+	// setInvalidFn: React.Dispatch<React.SetStateAction<boolean>>
```

Comment:
delete

Reply:
someone might want to see how it's done in the future, leave in for now